tests/other/metadce/hello_world_fastcomp_O3_MAIN_MODULE.sent

```diff
@@ -252,6 +252,38 @@ ___cxa_current_primary_exception
 ___cxa_decrement_exception_refcount
 ___cxa_end_catch
 ___cxa_find_matching_catch
+___cxa_find_matching_catch_0
```

Comment:
why are all these added?

Reply:
Hmm, I don't really know how the JS exceptions machinery works in dynamic linking across main module and side module - before this PR, what would happen if a side module throws an exception with say 3 arguments, but main module did not have a need to generate `___cxa_find_matching_catch_3` in code?

That is, superficially I get an impression that main module would need to emit all these variants to prepare for all possibilities in the side module?

Reply:
Honestly, I think exceptions probably did not work in all cases in dynamic linking, we only have very basic testing for that.